### ISSUE TITLE
KeyError: 'estimator'

### ISSUE THREAD

**Issue by ClemPalf** [awslabs/autogluon#1480]
Hello Innixma,

I tried with the pre-release version and by getting rid of the searcher parameter, unfortunately I still get the "KeyError: 'estimator'" error. Here is my updated notebook: https://github.com/ClemPalf/SageMaker-Inventory-monitoring/blob/main/2)%20Model_choice_with_autogluon.ipynb

Cheers, 

Clem

Ps: I tried with GPU-equipped instances but always ended up with workers from the data logger dying due to insufficient shared memory (I was using a ml.g4dn.xlarge). Do you have any tips on how to raise the shared memory limit on sagemaker notebook instances?

_Originally posted by @ClemPalf in https://github.com/awslabs/autogluon/issues/1474#issuecomment-1009746515_

**Comment by yinweisu** [comment]
Hi Clem,

I've tried to reproduce this issue on 1. a cpu ec2 instance 2. a gpu ec2 instance 3. a sagemaker cpu notebook, and I got no error on all of them. I cloned your repo, and hence was using the exact same data. On the sagemaker notebook, I directly cloned your repo and ran your example notebook. Maybe consider starting a new notebook and try again?

Regarding the sagemaker shared memory issue, it seems like there's no way to manipulate that and it is restricted to 64 megabytes. I would recommend launch up a gpu ec2 instance yourself (image classification with cpu is super slow).
